ftw/testbrowser/form.py

```diff
@@ -206,10 +206,8 @@ def submit(self, button=None):
                 return buttons.first.click()
 
         extra_values = None
-        if button and button.attrib.get('name', None) and \
-                button.attrib.get('value', None):
-            extra_values = {button.attrib['name']: button.attrib['value']}
-
+        if button and button.attrib.get('name', None):
+            extra_values = {button.attrib['name']: button.attrib.get('value', '')}
```

Comment:
yes this will send a empty string as value. well chrome sends it with content-disposition form-data with only a name and no content. which is represented in the plone request as an empty string.